### PR TITLE
Typescript example

### PR DESCRIPTION
**Pull request by xansec**
Add an example of a plugin written in Typescript

### CODE REVIEW

**Comment by xansec** [comment]
There were a couple of other plugin examples here I added, namely for gRPC translation. I also updated the README to be consistent and use the newer URLs.

File: plugins/README.md [hunk 1]
```diff
@@ -17,3 +21,16 @@ This example implements the same service as `python-auth-plugin` but written in
...
 OAuth OpenID Connect access token endpoint, implemented in Rust.
+This example implements the rewrite plugin adding an `Authorization` header to every request.
+
+### [typescript-rewrite-plugin](typescript-rewrite-plugin)
+
+This example shows how to use typescript as a plugin.
+
+### Response Classify Plugin Examples
+
+[Response Classify plugins](https://app.mayhem.security/docs/api-testing/guides/classify-plugin/) allow you to classify reponses recieved from the server with customer headers, status codes, or other data.
```
Comment (Ross-ForAllSecure) [inline plugins/README.md]:
```suggestion
[Response Classify plugins](https://app.mayhem.security/docs/api-testing/guides/classify-plugin/) allow you to classify responses received from the server with customer headers, status codes, or other data.
```

File: plugins/README.md [hunk 2]
```diff
@@ -17,3 +21,16 @@ This example implements the same service as `python-auth-plugin` but written in
...
+
+### [python-classify-plugin](python-classify-plugin)
+
+This example implements the classify plugin by parsing the body for grpc-statuses and using them to create HTTP reponses.
```
Comment (Ross-ForAllSecure) [inline plugins/README.md]:
```suggestion
This example implements the classify plugin by parsing the body for grpc-statuses and using them to create HTTP responses.
```

File: plugins/python-rewrite-plugin/README.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+source venv/bin/activate
+
+# Start a new fuzzing job named 'plugin-example' and run for 60 seconds
+mapi run --url <API_URL> plugin-example \
+  60 \
+  <API_SPECIFICATION_PATH> \
+  --rewrite-plugin src/plugin.py
+```
+
+If you wish to debug the server while `mapi` is running in your IDE of favorite
```
Comment (Ross-ForAllSecure) [inline plugins/python-rewrite-plugin/README.md]:
```suggestion
If you wish to debug the server while `mapi` is running in your IDE or favorite
```

**Review by Ross-ForAllSecure — APPROVED** [review]

**Comment by xansec** [comment]
Thanks!

**Pull request merged**